concepticondata/conceptlists/Koelle-1854-335.tsv

```diff
@@ -0,0 +1,336 @@
+Koelle-1854-335-90	90	Firewood	10	FIREWOOD
+Koelle-1854-335-91	91	Firewood (dry wood)	10	FIREWOOD
+Koelle-1854-335-92	92	Fish	227	FISH
+Koelle-1854-335-93	93	Five	493	FIVE
+Koelle-1854-335-94	94	Foot, or Instep of the Foot	1301	FOOT
+Koelle-1854-335-95	95	Foot-Sole	2666	SOLE (FOOT)
+Koelle-1854-335-96	96	Forehead	123	FOREHEAD
+Koelle-1854-335-97	97	Forest	420	FOREST
+Koelle-1854-335-98	98	Four	1500	FOUR
+Koelle-1854-335-99	99	Fourteen	2451	FOURTEEN
+Koelle-1854-335-100	100	Fowl (Hen)	265	FOWL
+Koelle-1854-335-101	101	Friend (My, Thy)	1325	FRIEND
+Koelle-1854-335-102	102	Frog	503	FROG
+Koelle-1854-335-103	103	Girl	1646	GIRL
+Koelle-1854-335-104	104	Goat	1502	GOAT
+Koelle-1854-335-105	105	God	1944	GOD
+Koelle-1854-335-106	106	Gold	1369	GOLD
+Koelle-1854-335-107	107	Gold (brass)		
+Koelle-1854-335-108	108	Good	1035	GOOD
+Koelle-1854-335-109	109	Grandfather	1383	GRANDFATHER
+Koelle-1854-335-110	110	Grandmother	1496	GRANDMOTHER
+Koelle-1854-335-111	111	Great, large	1202	BIG
+Koelle-1854-335-112	112	Greedy	2018	GREEDY
+Koelle-1854-335-113	113	Greegree	2804	AMULET
+Koelle-1854-335-114	114	Ground-nut	595	GROUNDNUTS
+Koelle-1854-335-115	115	Guinea-Corn (bearing like Maize)		
+Koelle-1854-335-116	116	Gullet		
+Koelle-1854-335-117	117	Gun	1566	GUN
+Koelle-1854-335-118	118	Hair	1040	HAIR
+Koelle-1854-335-119	119	Hat	771	HAT
+Koelle-1854-335-120	120	Head	1256	HEAD
+Koelle-1854-335-121	121	Heaven (sky)	1565	HEAVEN
+Koelle-1854-335-122	122	Heel	980	HEEL
+Koelle-1854-335-123	123	Hell	878	HELL
+Koelle-1854-335-124	124	Hoe	284	HOE
+Koelle-1854-335-125	125	Honey	942	HONEY
+Koelle-1854-335-126	126	Horse	615	HORSE
+Koelle-1854-335-127	127	Hot	1286	HOT
+Koelle-1854-335-128	128	House	1252	HOUSE
+Koelle-1854-335-129	129	I bathe (wash myself)	138	BATHE
+Koelle-1854-335-130	130	I beg	3534	BEG
+Koelle-1854-335-131	131	I break a stick	2558	BREAK (CLEAVE)
+Koelle-1854-335-132	132	I breathe	1407	BREATHE
+Koelle-1854-335-133	133	I buy	1869	BUY
+Koelle-1854-335-134	134	I call a slave		
+Koelle-1854-335-135	135	I catch a fish	2638	FISHING
+Koelle-1854-335-136	136	I come	1446	COME
+Koelle-1854-335-137	137	I cook meat		
+Koelle-1854-335-138	138	I cough	879	COUGH
+Koelle-1854-335-139	139	I cover a pot		
+Koelle-1854-335-140	140	I cut a tree	3151	CUT (WITH AXE)
+Koelle-1854-335-141	141	I dance	1879	DANCE
+Koelle-1854-335-142	142	I die	1494	DIE
+Koelle-1854-335-143	143	Idol	1945	IDOL
+Koelle-1854-335-144	144	I do not dance		
+Koelle-1854-335-145	145	I do not play		
+Koelle-1854-335-146	146	I dream	1920	DREAM (SOMETHING)
+Koelle-1854-335-147	147	I drink water		
+Koelle-1854-335-148	148	I eat corn		
+Koelle-1854-335-149	149	I eat rice		
+Koelle-1854-335-150	150	I eat yam		
+Koelle-1854-335-151	151	I eat yams		
+Koelle-1854-335-152	152	I fall	2894	TUMBLE (FALL DOWN)
+Koelle-1854-335-153	153	I flog a child		
+Koelle-1854-335-154	154	I give thee	1447	GIVE
+Koelle-1854-335-155	155	I go	695	GO
+Koelle-1854-335-156	156	I hear	1408	HEAR
+Koelle-1854-335-157	157	I kill a fowl		
```

Comment:
SLAUGHTER?

Reply:
I suppose we could map it... It is not entirely clear from the data if in all cases it is a "single" entity, but the case is analogous to "catch a fish"...